src/rand/rand.zig

```diff
@@ -114,6 +114,66 @@ pub fn BlockRng(
+        67349919375139501,  170711730505131607, 144992818219067542, 30670516022729165,  138907584615587438, 127817381089378212, 114871171103897336, 165002752098828949, 129783111060717438, 141402390895707373, 49789905082632543,  44642008510651678,  142897357041335417, 15619830375298759,  108389393594576866, 2049459598661566,   186002123029450793, 115978668653229962, 25505487210171515,  5594356300076705,   218194016043222958, 131099199892564490, 161829150048369624, 107822998318207832, 97744594758044930,  159723516838411175, 137761953409828195, 17934832723300816,  199618139361605350, 152488795447658825, 59285081064243544,  189888454568762638, 214109977383678979, 106069302482628549, 125828040156667548, 60448978859164630,  6664155818271300,   32057213396689758,  134359424014350909, 101706944218512714, 34363235225823814,  224815739117243291, 154174716140977289, 186508220141956340, 81371844716237169,  209604334597774355, 69722560753843310,  26554189446811940,  112142350402187528, 55025340774553552,
+        150245030477280661, 213624595435805867, 98539226549693351,  33801042929801430,  203768025145824241, 85942546992148859,  75920971919241723,  178282653246684682, 107180956559443505, 5383407800161210,   25924734343585423,  113505140769538087, 158294491487555208, 202058441263127158, 23754109538483632,  164332741174520337, 177386067656078943, 63092035208918209,  105193165892558811, 90842259062651642,  200087190942123447, 118864219926511175, 178914468731322740, 123929446491278238, 83233452188148132,  130678632301769656, 148140195893539773, 189717099709818244, 3866162913691558,   165276931946696165, 195669972620483101, 220740178411808744, 218597519430516114, 181499314628684163, 56281591694254078,  112584340323866656, 73160071773275256,  216001312005955998, 184061143539899962, 132782144182150573, 40566523540155677,  171562338978082216, 151758301674069632, 173398251098334249, 63757156897079302,  81032160488073578,  173298561206951917, 154421360720272045, 15981567533895039,  88393403508710277,
+    };
```

Comment:
Might be a good idea to move this to its own file, seems a bit out of place in the middle of everything. Ie something like `@import("test_data.zig").rust_chacha_uint_less_than_sequence`.
Rationale:
 * If it's horizontally longer, it's quicker to scroll past, but it causes the editor side scroll to be huge, which is awkward.
 * If it's vertically longer, it takes a long time to move past and breaks the flow of code.
 * Always best to keep test data separate from tests if it's meaningful and specific like this.

Another option would be to generate a file for this in build.zig that could then be obtained with ie `@embedFile("rust_chacha_uint_less_than_sequence")` - I leave that decision to you, though feel free to reach out if you want to do that but are unsure how.

Reply:
I agree, for now I have just moved it to the base of the file and adjusted width. I think this is something that we should come up with a plan of attack for the project as a whole and then be consistent.

[060375](https://github.com/Syndica/sig/pull/211/commits/060375c68e4b0d08b9f6b52c6a2047f9ae00765c)